Mathlib/RingTheory/HopfAlgebra/TensorProduct.lean

```diff
@@ -13,29 +13,39 @@ We define the Hopf algebra instance on the tensor product of two Hopf algebras.
 
 -/
 
-universe u
-namespace TensorProduct
-variable {R A B : Type u} [CommRing R] [Ring A] [Ring B] [HopfAlgebra R A] [HopfAlgebra R B]
-
-open HopfAlgebra
-
-noncomputable instance instHopfAlgebra : HopfAlgebra R (A ⊗[R] B) :=
-  { antipode := map antipode antipode
-    mul_antipode_rTensor_comul := by
-      simp only [comul_def, counit_def, LinearMap.rTensor_def,
-        ← map_id, ← LinearMap.comp_assoc (map _ _), ← tensorTensorTensorComm_comp_map,
-        Algebra.mul'_comp_tensorTensorTensorComm, ← map_comp]
-      simp only [← LinearMap.rTensor_def, LinearMap.comp_assoc, mul_antipode_rTensor_comul,
-        map_comp, Algebra.linearMap_comp_mul']
-    mul_antipode_lTensor_comul := by
-      simp only [comul_def, counit_def, LinearMap.lTensor_def,
-        ← map_id, ← LinearMap.comp_assoc (map _ _), ← tensorTensorTensorComm_comp_map,
-        Algebra.mul'_comp_tensorTensorTensorComm, ← map_comp]
-      simp only [← LinearMap.lTensor_def, LinearMap.comp_assoc, mul_antipode_lTensor_comul,
-        map_comp, Algebra.linearMap_comp_mul'] }
+open TensorProduct Coalgebra.TensorProduct
+
+namespace HopfAlgebra.TensorProduct
```

Comment:
```suggestion
namespace TensorProduct
```
I don't think you particularly want `antipode_def` in the nested namespace (and you'd need to deprecate it if you did)